CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## 1.0.1
+* Fixed incorrectly location of axises titles
```

Comment:
1.0.1 was released yesterday. Please don't include any extra release notes. Please update version of the visual if you want to describe release notes.